stellarphot/photometry/tests/test_profiles.py

```diff
@@ -171,7 +171,7 @@ def test_radial_profile_with_background(self):
             distribution="gaussian",
             mean=100,
             stddev=noise_stdev,
-            seed=43917,
+            seed=20250404,
```

Comment:
Did the seed have to be a larger number to work with the newer version of photutils?

Reply:
So you are just increasing the simulated amount of noise to be closer to "real" values?